PsychSourceGL/Cohorts/FTGLTextRenderer/libptbdrawtext_ftgl.cpp

```diff
@@ -450,11 +449,7 @@ void PsychSetTextBGColor(int context, double* color)
 // renderbackend needs a different geometric setup:
 void PsychSetTextViewPort(int context, double xs, double ys, double w, double h)
```

Comment:
That revert is incomplete. You don't set the projection matrix based on _vxs et al.